app/presenters/card_presenter.rb

```diff
@@ -128,6 +128,17 @@ def output_icon
     raise 'missing card header icon'
   end
 
+  def card_style
+    unless %w[show edit].include?(action_name) &&
+           instance.present? &&
+           instance.respond_to?(:active?) &&
```

Comment:
should we remove the `instance.respond_to?(:active?)` and just require that we always implement `active?` on every model so we never forget any of these?